### ISSUE TITLE
Unable to view Docker images tags inside portus

### ISSUE THREAD

**Issue by arjunm183** [SUSE/Portus#1009]
Hello Team,

I am unable to view the tags for the docker images inside my portus.

Looks like a bug. Can you help me with thi s.

Portus Version : rpm -qa | grep portus
portus-20160523081115-1.1.x86_64

Operating System : SLES 12 SP1

Screenshots for error as Below.
![image](https://cloud.githubusercontent.com/assets/5314336/17164371/20c8f596-53e5-11e6-83db-fb51b9d15017.png)

In the below image you can see the error when i click on the docker image in portus.
![image](https://cloud.githubusercontent.com/assets/5314336/17164378/3791b5ba-53e5-11e6-8fa5-12dd669ba063.png)

Regards,
Arjun.M

**Comment by mssola** [comment]
Hi @arjunm183, since you are using the RPM, could you provide what does `portusctl portus:info` give to you. Moreover, logs should be helpful here, simply execute `portusctl logs`.

Note also that your version is quite old, and that you should be able to upgrade from [OBS](https://build.opensuse.org/project/show/Virtualization:containers:Portus:2.0). Maybe it could be useful to you to upgrade and try to reproduce this issue.

**Comment by arjunm183** [comment]
Output for the below command 

 portusctl portus:info
Could not find command "portus:info"

[portus_logs.tar.gz](https://github.com/SUSE/Portus/files/389010/portus_logs.tar.gz)

Attached is the logs .

I am downloading the new rpm and try to install it and check if the issue occurs

**Comment by arjunm183** [comment]
Fixed the issue by downgrading Portus which is available in OBS.

**Comment by arjunm183** [comment]
Or i am not sure its because of portus crono catalog job.

what does this portus crono does?

**Comment by mssola** [comment]
> Fixed the issue by downgrading Portus which is available in OBS.

By downgrading you meant upgrading right ?

> what does this portus crono does?

It keeps the database of Portus in sync with the registry just in case it became out-of-date due to some networking error. Check [this page on the documentation](http://port.us.org/features/1_Synchronizing-the-Registry-and-Portus.html) for more info.

**Comment by arjunm183** [comment]
Hi @mssola I have downgraded it and fixed it , however i face issues like Portus often hungs. I have to restart apache2 often to make the docker pull happening.

Can you recomment a proper version of Portus and Docker registry to use from SUSE

**Comment by mssola** [comment]
> Hi @mssola I have downgraded it and fixed it , however i face issues like Portus often hungs. I have to restart apache2 often to make the docker pull happening.

Downgraded to which version ? The hangs are a known problem for configurations with too few workers. See [the comments on this issue](https://github.com/SUSE/Portus/issues/373) for more information.

> Can you recomment a proper version of Portus and Docker registry to use from SUSE

I'd recommend Portus 2.0.5 and Docker Registry 2.5. That is, the latest versions of both applications. This should work fine in openSUSE.

**Comment by vitoravelino** [comment]
Is this issue still valid, @mssola? Currently, in the repositories page we don't show any tag, the user needs to go to the repository specific page. I know that there's an error reported but the latest version at this moment was 2.0.5.

**Comment by mssola** [comment]
Agree with @vitoravelino. Re-open this issue if this is still valid in master. Thanks for reporting 👍

**Comment by ITPROMatthew** [comment]
I think this should be re-opened.

I am experiencing this on `:head`

```
Started GET "/repositories/1" for 10.2.192.2 at 2017-11-08 01:51:02 +0000
Processing by RepositoriesController#show as HTML
  Parameters: {"id"=>"1"}
  Rendered repositories/show.html.slim within layouts/application (107.1ms)
Completed 500 Internal Server Error in 210ms (ActiveRecord: 31.7ms)

ActionView::Template::Error (undefined method `[]' for nil:NilClass):
    55:
    56:   .panel-body
    57:     <loading-icon v-if="isLoading"></loading-icon>
    58:
    59:     .table-responsive.tags
    60:       <tags-not-loaded v-if="notLoaded"></tags-not-loaded>
    61:       <tags-table v-if="!isLoading && !notLoaded" :tags="tags" :can-destroy="#{can_destroy?(@repository)}" :state="state" :security-enabled="#{security_vulns_enabled?}"></tags-table>
  lib/portus/security_backends/base.rb:25:in `configuration'
  lib/portus/security_backends/base.rb:18:in `enabled?'
  lib/portus/security.rb:21:in `block in initialize'
  lib/portus/security.rb:21:in `each'
  lib/portus/security.rb:21:in `initialize'
  app/helpers/repositories_helper.rb:134:in `new'
  app/helpers/repositories_helper.rb:134:in `security_vulns_enabled?'
  app/views/repositories/show.html.slim:58:in `_app_views_repositories_show_html_slim__2050362486129622288_43793260'
  app/controllers/repositories_controller.rb:19:in `show'


Processing by ErrorsController#show as HTML
  Parameters: {"id"=>"1", "status"=>"500"}
  Rendered errors/500.html.erb within layouts/errors (1.6ms)
  Rendered errors/_status_title.html.slim (31.1ms)
Completed 500 Internal Server Error in 60ms (Views: 57.7ms | ActiveRecord: 0.0ms)
```

**Comment by mssola** [comment]
Fixed in #1526 if I'm not mistaken 😃
